### PR TITLE
Create mint-farm-yusdbtc.md

### PR DESCRIPTION
**Pull request by henrystats**
Updating tutorial guide for minting yusdbtc

### CODE REVIEW

File: docs/tutorials/mint-farm-yusdbtc.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+$1 by the time of expiry and you will lose
+money if you are buying `yUSD` above $1.
+
+:::
+
+By supplying liquidity to the `yUSD/USDC` Balancer pool, you have the
+opportunity to earn `BAL`, `REN` and `UMA` tokens. The `UMA` token distribution
+is governed by the experimental UMA liquidity mining program.
+
+[INSERT LINK TO RENBTC LIQUIDITY MINING PROGRAM BLOG POST]
```
Comment (chrismaree) [inline docs/tutorials/mint-farm-yusdbtc.md:125]: I think this link is breaking CI. This is the output from Vercel:
image

**Review by chrismaree — COMMENTED** [review]
This is awesome! thanks so much @henrystats for putting in the effort and writing such a great post.

**Review by chrismaree — APPROVED** [review]
This is awesome! thanks for the hard work on this PR. Sorry about all the difficulty of getting it through CI.

**Comment by henrystats** [comment]
> This is awesome! thanks for the hard work on this PR. Sorry about all the difficulty of getting it through CI.

Cheers! thank you very much for your help as well!

File: docs/tutorials/mint-farm-yusdbtc.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+![amount](https://github.com/henrystats/docs/blob/master/static/img/renBTC%20guide%202.png)
+
+then, click `Next` 
+
+![next](https://github.com/henrystats/docs/blob/master/static/img/renBTC%20guide%204.png)
+
+then click `confirm`. Before doing this, make sure you verify all the details in the modal.
+
+Check that you are minting(wrapping) BTC, as Renbridge allows you to wrap other coins like Zen and BCH
```
Comment (claytonroche) [inline docs/tutorials/mint-farm-yusdbtc.md:42]:
```suggestion
Check that you are wrapping Bitcoin, as Renbridge allows you to wrap other coins like Zen and BCH.
```

File: docs/tutorials/mint-farm-yusdbtc.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+`Destination` should be your Metamask wallet address, this is where your renBTC will be sent to.
```
Comment (claytonroche) [inline docs/tutorials/mint-farm-yusdbtc.md:44]:
```suggestion
The `Destination` should be your Metamask wallet address, this is where your renBTC will be sent to.
```

File: docs/tutorials/mint-farm-yusdbtc.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+Also check the cost (`RenVM fee`, `Bitcoin Network fee`) and the amount of renBTC you will be receiving.
```
Comment (claytonroche) [inline docs/tutorials/mint-farm-yusdbtc.md:46]:
```suggestion
Also, check the cost (`RenVM fee`, `Bitcoin Network fee`) and the amount of renBTC you will be receiving.
```

File: docs/tutorials/mint-farm-yusdbtc.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+
+Once you have done all of this, you can then go ahead and click `confirm`.
+
+![confirm](https://github.com/henrystats/docs/blob/master/static/img/renBTC%20guide%203.png)
+
+A one-time deposit address is generated for you to send your BTC to. Send the exact amount of BTC you are wrapping to that address, in a single transaction.
+
+![deposit](https://github.com/henrystats/docs/blob/master/static/img/Screenshot%20(142).png)
+
+* Your transaction requires 6confirmations(can take up to 1hour) before renBTC is sent to your wallet
```
Comment (claytonroche) [inline docs/tutorials/mint-farm-yusdbtc.md:56]:
```suggestion
* Your transaction requires 6 confirmations, which can take up to 1 hour, before the renBTC is sent to your wallet 
```

File: docs/tutorials/mint-farm-yusdbtc.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+* Your transaction requires 6confirmations(can take up to 1hour) before renBTC is sent to your wallet 
+* You will need to accept & sign the transaction that sends renBTC to your wallet.
+
+
+
+## Step 2: Minting yUSD with renBTC
+
+Minting yUSD tokens is easy with the [EMP Tools](http://tools.umaproject.org/),
+UMA's UI for ExpiringMultiParty contracts (EMP) which is what the yUSD token is
+minted from.
```
Comment (claytonroche) [inline docs/tutorials/mint-farm-yusdbtc.md:65]:
```suggestion
```

File: docs/tutorials/mint-farm-yusdbtc.md [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+
+* Your transaction requires 6confirmations(can take up to 1hour) before renBTC is sent to your wallet 
+* You will need to accept & sign the transaction that sends renBTC to your wallet.
+
+
+
+## Step 2: Minting yUSD with renBTC
+
+Minting yUSD tokens is easy with the [EMP Tools](http://tools.umaproject.org/),
+UMA's UI for ExpiringMultiParty contracts (EMP) which is what the yUSD token is
```
Comment (claytonroche) [inline docs/tutorials/mint-farm-yusdbtc.md]:
```suggestion
UMA's UI for ExpiringMultiParty contracts (EMP), of which the uUSD token is.
```

File: docs/tutorials/mint-farm-yusdbtc.md [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+## Step 3: Farming uUSDrBTC on Balancer
+
+:::danger
+
+Supplying _only_ `USDC` to the pool is very dangeorus and is not recommended,
+because you are essentially buying `uUSDrBTC` on the open market. You should avoid
+this because `uUSDrBTC` will be worth
+$1 by the time of expiry and you will lose
+money if you are buying `uUSDrBTC` above $1.
+
```
Comment (claytonroche) [inline docs/tutorials/mint-farm-yusdbtc.md:118]: Change the above warning to say instead:

Supplying _only_ 'USDC' to the pool will result in auto-purchasing uUSD at the current price.  This means you are market-buying uUSDrBTC, so be mindful of the trading price of uUSDrBTC before doing this.

File: docs/tutorials/mint-farm-yusdbtc.md [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+:::
+
+By supplying liquidity to the `uUSDrBTC/USDC` Balancer pool, you have the
+opportunity to earn `BAL`, `REN` and `UMA` tokens. The `UMA` token distribution
+is governed by the experimental [UMA liquidity mining program](https://medium.com/uma-project/uma-announcing-the-yield-dollar-on-renbtc-440a1ed0c5d5).
+
```
Comment (claytonroche) [inline docs/tutorials/mint-farm-yusdbtc.md:124]: Change the last scentence above to say:

Read about this liquidity mining program here in the [relevant announcement](https://medium.com/uma-project/uma-announcing-the-yield-dollar-on-renbtc-440a1ed0c5d5).

File: docs/tutorials/mint-farm-yusdbtc.md [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+be asked to Setup Proxy if you haven’t already
+
+![add liquidity button on balancer](/docs/tutorials/uUSD_addliquiditybal.png)
+
+Once the proxy is setup, the "Add Liquidity" button will show you a form for
+adding liquidity.
+
+At this point, you might want to get some `USDC` so you can supply both assets
+to the Balancer pool. Alternatively, Balancer also supports supplying only a
+single-asset but that mechanism is out of scope for this tutorial.
```
Comment (claytonroche) [inline docs/tutorials/mint-farm-yusdbtc.md:136]:
```suggestion
single-asset, which will auto-purchase the complementary asset.
```

**Pull request merged**